source/about/corporate-directory-integration.rst

```diff
@@ -2,7 +2,7 @@
 Corporate Directory Integration 
 ================================
 
-Mattermost Enterprise Edition offers advanced security and authentication options for integrating with corporate directories. This guide outlines the options available.
+Mattermost offers advanced security and authentication options for integrating with corporate directories. This guide outlines the options available.
```

Comment:
Would we want to clarify it's for Enterprise Edition only?

Reply:
I chose to remove the Edition because there are many security features that are not explicit to Enterprise Edition only on this doc.  This outlines the various options for authentication, secure access, and Corporate Directory Integration.

Reply:
Got it